src/rules/completedDocsRule.ts

```diff
@@ -395,11 +396,42 @@ class CompletedDocsWalker extends Lint.ProgramAwareRuleWalker {
     }
 
     public visitVariableDeclaration(node: ts.VariableDeclaration): void {
-        this.checkNode(node, ARGUMENT_VARIABLES);
+        this.checkVariable(node);
         super.visitVariableDeclaration(node);
     }
 
-    private checkNode(node: ts.NamedDeclaration, nodeType: DocType, requirementNode: ts.Declaration = node): void {
+    private checkVariable(node: ts.VariableDeclaration) {
+        // Only check variables in variable declaration lists
+        // and not variables in catch clauses and for loops.
+        const list = node.parent!;
+        if (!isVariableDeclarationList(list)) {
+            return;
+        }
+
+        const statement = list.parent!;
+        if (!isVariableStatement(statement)) {
+            return;
+        }
+
+        // Only check variables at the namespace/module-level or file-level
+        // and not variables declared inside functions and other things.
+        let check = false;
```

Comment:
Doh! Sorry, missed that on your previous comment. Am I correct in assuming it's defined as `parent?: Node` because it _can_ be undefined if the `setParentNodes` argument is `false` when you load the TypeScript source file. But since TSLint always passes `true` to that argument, the parent nodes will always be set (except on the root `SourceFile` node)?